### ISSUE TITLE
[LvlUp] Unexpected keyword argument 'mention_author'

### ISSUE THREAD

**Issue by EternalllZM** [vertyco/vrt-cogs#182]
**Describe the bug**
Provided details over direct messages. Essentially, the slashtag version of the command has an issue when running the equivalent of `[p]pf` that is related to changes made to the cog recently.

**Cog Version***
Was using the very latest 3.15.6, have now downgraded to commit a453411 which is version 3.14.5 where this issue does not occur.

**To Reproduce**
1. Create provided slashtag
2. Attempt to run the slash command with no further arguments
3. Command will fail and error will populate.

**Expected behavior**
Should run like before without issue.

**Error**
```
Traceback (most recent call last):
  File "/home/red/redenv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/red/.local/share/Red-DiscordBot/data/modet/cogs/CogManager/cogs/levelup/common/base.py", line 815, in get_profile
    await self.send_user_profile(user, ctx.channel, message=ctx.message)
  File "/home/red/.local/share/Red-DiscordBot/data/modet/cogs/CogManager/cogs/levelup/common/base.py", line 979, in send_user_profile
    await send(em, mention)
  File "/home/red/.local/share/Red-DiscordBot/data/modet/cogs/CogManager/cogs/levelup/common/base.py", line 851, in send
    await message.reply(embed=obj, mention_author=mention, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/red/.local/share/Red-DiscordBot/data/modet/cogs/CogManager/cogs/slashtags/objects.py", line 722, in reply
    return send(content, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Webhook.send() got an unexpected keyword argument 'mention_author'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/red/redenv/lib/python3.11/site-packages/discord/ext/commands/bot.py", line 1350, in invoke
    await ctx.command.invoke(ctx)
  File "/home/red/redenv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/red/redenv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: Webhook.send() got an unexpected keyword argument 'mention_author'
```

**Comment by EternalllZM** [comment]
This might be something that needs to be updated in slashtags, if the lvlup code is correct and the slashtags is not. However, I have not experienced this issue with any other slashtag command before.

**Comment by vertyco** [comment]
attempt #1 to fix this has been pushed

**Comment by vertyco** [comment]
fixed in v3.15.8
